services/intersection-api/api/.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
It looks like the project comes with the maven wrapper, but commands reference mvn instead of mvnw. Is this maven wrapper being used at all?

Reply:
I need some guidance on this - I would normally just use mvn, but I got a few comments from Matt requesting to add the mvnw wrapper files (unless I misunderstood). What approach would you recommend here @mcook42?